x-pack/plugins/integration_assistant/server/util/cel_input.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CelInputStateDetails } from '../graphs/cel/types';
```

Comment:
If this file will only be used in graphs , then you may move this under graphs?

Reply:
I can certainly do that. I was torn initially where to put it.

Reply:
Also renamed